pkg/webui/console/components/webhook-form/index.js

```diff
@@ -208,17 +223,29 @@ export default class WebhookForm extends Component {
   modalResolve = () => null
   modalReject = () => null
 
-  state = {
-    error: undefined,
-    displayOverwriteModal: false,
-    existingId: undefined,
+  constructor(props) {
+    super(props)
+    const { initialWebhookValue } = this.props
+
+    this.state = {
+      error: undefined,
+      displayOverwriteModal: false,
+      existingId: undefined,
+      mayShowCredentialsInput: initialWebhookValue.headers.Authorization?.startsWith('Basic'),
```

Comment:
```suggestion
      mayShowCredentialsInput: initialWebhookValue.headers.Authorization?.startsWith('Basic'),
```

Appears to me like this will error if `Authorization` evaluates to `undefined`.